src/main/java/com/starbucks/service/OrderService.java

```diff
@@ -18,39 +17,31 @@ public OrderService(OrderRepository orderRepository) {
     }
 
     public String saveOrder(Order order) {
-
-        orderRepository.save ( order );
-        return order.toString ( );
+        orderRepository.save(order);
```

Comment:
it should be something like return orderRepository.save(order); save method is supposed to return the object it saves, but don't to this toString, it annoys me a lot )))